db/seeds.rb

```diff
@@ -1,8 +1,8 @@
 Article.destroy_all
 Category.destroy_all
 
-Category.create(name: 'Weather')
-Category.create(name: 'Sport')
+Category.create(name: 'Weather', id: 1)
+Category.create(name: 'Sport', id: 2)
```

Comment:
Why do you feel that you need to set the `id` attribute in your seed?

Reply:
Because if we rerun db:seed without dropping the database the id will increase, and in the create article method we link them to these exact ids. 
That's why we hard-coded it